subprojects.cfg

```diff
@@ -32,7 +32,7 @@ firebase-datatransport
 firebase-dynamic-links
 firebase-dynamic-links:ktx
 firebase-firestore
-firebase-firestore:ktx
+# firebase-firestore:ktx
```

Comment:
is this comment supposed to be included in this PR?

Reply:
Yeah. Normally, this change would require some changes in the `ktx` package- but since those will be deleted very soon, I opted to just comment it out to avoid polluting the diff.